public/app/themes/clarity/inc/admin/plugins/wp-all-export.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace MOJ\Intranet;
+
+defined('ABSPATH') || exit;
+
+/**
+ * How to filter the wp-all-export output for stakeholder export requests.
+ * 
+ * - Import an upto date copy of the production to localhost.
+ * - Set memory_limit=512M for the local/php-fpm.php.
+ * - Install wp-all-export locally.
+ * - Disable WP Media Offload Plugin - so urls are not filtered.
+ * - Include this file in functions.php.
+ * - Set the export WP_Query to: 'post_type' => 'page', 'post_status' => 'publish'.
+ * - Set: Fields ID, Title, Content, URL, MenuOrder, Date.
+ * - Increase the batch size so that a single batch covers all results, this allows us to sort by URL.
+ * - Download the csv.
+ * - Check local urls in content work. Search for 'docker'.
+ * - Deactivate and uninstall wp-all-export.
+ */
+
+class WpAllExport
+{
+
+    public function __construct()
+    {
+        add_filter('wp_all_export_csv_rows', [$this, 'customOperationsAreaFilter']);
+    }
+
+    /**
+     * This is an example of a customised function to filter the output oft the wp-all-export plugin.
+     * 
+     * @param array $articles the rows being exported.
+     * 
+     * @return array the filtered and mapped rows.
+     */
+
+    public function customOperationsAreaFilter($articles)
+    {
+
+        $parent_id = 69769;
```

Comment:
There's a hard-coded number here, is this expected?

Reply:
Good spot :) I've replaced with an env var, and updated the instructions.